openpos-flow/src/main/java/org/jumpmind/pos/core/ui/ActionItem.java

```diff
@@ -50,6 +50,8 @@ public class ActionItem implements Serializable {
     protected String imageUrl;
     @Builder.Default
     protected boolean enabled = true;
+    @Builder.Default
+    protected boolean iconMuted = false;
```

Comment:
Primitive booleans default to false automatically. You don't need this or the builder.default
```suggestion
    protected boolean iconMuted;
```

Reply:
right. Thanks.